style.css

```diff
@@ -575,6 +575,102 @@ li {
   top: -21.5%;
 }
 
+.get-in-touch {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 120px 24px;
+}
+
+.get-in-touch-form > * {
+  padding: 0;
+  margin-bottom: 36px;
+}
+
+/* .required::after{
+  content: "*";
+  color: red;
+} */
```

Comment:
- [ ] Please remove commented line of code. Commented line of code make project look dirty and unfinished. Removing this will make your project more clean 🧼 and professionnal 😄

Reply:
Resolved Required Changes

- Removed the blank line and spaces inside the <textarea>.

- Button interaction like hover added

- Already contact form submission is working perfectly when you fill out all forms and then click on submit button it works perfectly.

- Removed commented line of code